test/unit/cpu/CMakeLists.txt

```diff
@@ -61,21 +60,18 @@ if (RAJA_ENABLE_CUDA)
   endif()
 
   cuda_add_executable(indexset-tests
-    test-indexsets.cxx
-    buildIndexSet.cxx)
+    test-indexsets.cxx)
 
   cuda_add_executable(traversal-tests
-    test-traversals.cxx
-    buildIndexSet.cxx)
+    test-traversals.cxx)
 
 else()
+  cuda_add_library(bis buildIndexSet.cxx)
```

Comment:
It certainly should.  Thanks!

On 26 Jul 2016, at 11:06, David Beckingsale wrote:

>>
>>  else()
>> +  cuda_add_library(bis buildIndexSet.cxx)
>
> @trws the build is failing - this should just be `add_library`.
>
> ---
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub:
> https://github.com/LLNL/RAJA/pull/100/files/2d2ea4aa496b46421f094e3d5e85d9de7f491e88#r72304667